fedimint-server/src/consensus/engine.rs

```diff
@@ -639,10 +639,11 @@ impl ConsensusEngine {
 
         debug!(%peer, item = ?DebugConsensusItem(&item), "Processing consensus item");
 
-        self.last_ci_by_peer
-            .write()
-            .await
-            .insert(peer, session_index);
+        self.ci_status_senders
+            .get(&peer)
+            .expect("No ci status sender for peer {peer}")
+            .send(Some(session_index))
+            .ok();
```

Comment:
`.ok()` is not OK :P

If it should never happen, please add `warn`, so when debugging the reviewer can be sure it didn't happen. :)